financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/AccessibleDataCallout.kt

```diff
@@ -141,16 +141,21 @@ private fun AccessibleDataText(
     val permissionsReadable = remember(model.permissions) { model.permissions.toStringRes() }
     AnnotatedText(
         text = TextResource.StringId(
-            value = when (model.isStripeDirect) {
-                true -> when (model.businessName) {
-                    null -> R.string.data_accessible_callout_through_stripe_no_business
-                    else -> R.string.data_accessible_callout_through_stripe
+            value = when {
+                model.isNetworking -> when (model.businessName) {
+                    null -> R.string.data_accessible_callout_through_link_no_business
+                    else -> R.string.data_accessible_callout_through_link
                 }
 
-                false -> when (model.businessName) {
+                model.isStripeDirect -> when (model.businessName) {
```

Comment:
for iOS there is no conditional around businessName for isStripeDirect

Reply:
not sure if it can happen, but let's leave it just in case? 😄

Reply:
https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/8bd67461/src/linkedAccounts/shared-ui/DataAccessText/MerchantDataAccessText.tsx#L38-L39

Reply:
oh! yeah this is wrong then. Updating it, thanks!

Reply:
https://github.com/stripe/stripe-android/pull/6375/commits/52c6b668360c2ed91dd68c96a96b561164da3630